agent/health/health.go

```diff
@@ -67,6 +67,9 @@ type Probes []*pb.Probe
 // Add adds a health probe for a specific node.
 // Implements Reporter
 func (r *Probes) Add(probe *pb.Probe) {
+	if probe.Status == pb.Probe_Failed && probe.Severity == pb.Probe_Cleared {
```

Comment:
Not sure I understand this logic. Could you explain why severity is being adjusted here, maybe add a comment?

Is this to set severity on probes that do not set it themselves?

Reply:
It was meant as a refactoring aid to match the old behavior - whenever a failed probe is added w/o an explicitly set severity, it is reset to `Critical`.